Makefile

```diff
@@ -280,7 +280,7 @@ check-contrib:
 	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/semconv=$(CURDIR)/semconv"
 	@$(MAKE) -C $(CONTRIB_PATH) for-all CMD="$(GOCMD) mod edit -replace go.opentelemetry.io/collector/service=$(CURDIR)/service"
 	@$(MAKE) -C $(CONTRIB_PATH) -j2 gotidy
-	@$(MAKE) -C $(CONTRIB_PATH) test
+	@$(MAKE) -C $(CONTRIB_PATH) gotest
```

Comment:
This makes me think we should change the `test` target in the contrib repo perhaps....

Reply:
I don't know about that, `make test` is defined in `Makefile.Common` so each component can run tests, and `make gotest` is only defined in root. Other tasks follow the same pattern, `gotidy` vs `tidy` for example.

Reply:
Yeah that's fair... test is just more intuitive than gotest for anyone starting with the repo